packages/migrate/loaderOptionsPlugin/loaderOptionsPlugin.ts

```diff
@@ -37,8 +37,8 @@ export default function(j: IJSCodeshift, ast: INode): INode {
 			});
 	}
 
-	// If there is UglifyJsPlugin, set minimize: true
-	if (findPluginsByName(j, ast, ["webpack.optimize.UglifyJsPlugin"]).size()) {
+	// If there is TerserPlugin, set minimize: true
+	if (findPluginsByName(j, ast, ["webpack.optimize.TerserPlugin"]).size()) {
```

Comment:
webpack has no export on `webpack.optimise.TerserPlugin`. The old code has to be kept. We want to move from `webpack.optimise.UglifyJsPlugin` on `plugin[]` to `optimize.minimizer[]` then add terser.

Reply:
yes. So @abenezerabebe you will need to add a require statement and use `TerserPlugin()` instead

Reply:
@evenstensberg I've replace lookups for `TerserPlugin` in `webpack.optimise` with requiring `terser-webpack-plugin` and using TerserPlugin itself. Do I need to do anything concerning `optimize.minimizer[]`?